tests/protocol_arg.pyi

```diff
@@ -0,0 +1,7 @@
+from typing import Protocol
+
+class P(Protocol):
+    def method1(self, arg: int) -> None: ...  # Y062 Argument "arg" to protocol method "method1" should not be positional-or-keyword (suggestion: make it positional-only)
+    def method2(self, arg: str, /) -> None: ...
+    def method3(self, *, arg: str) -> None: ...
+    def method4(self, __arg: int) -> None: ...
```

Comment:
not really necessary in terms of code coverage, but we could maybe also add some examples that use both positional-only and keyword-only args, for completeness:

```suggestion
    def method4(self, __arg: int) -> None: ...
    def method5(self, __arg: int, *, foo: str) -> None: ...
    def method6(self, arg, /, *, foo: str) -> None: ...
```